src/strategy/dungeons/wotlk/forgeofsouls/FosMultipliers.h

```diff
@@ -0,0 +1,24 @@
+#ifndef _PLAYERBOT_WOTLKDUNGEONFOSMULTIPLIERS_H
+#define _PLAYERBOT_WOTLKDUNGEONFOSMULTIPLIERS_H
+
+#include "Multiplier.h"
+
+class BronjahmMultiplier : public Multiplier
+{
+    public:
+    BronjahmMultiplier(PlayerbotAI* ai) : Multiplier(ai, "bronjahm") {}
+
+    public:
+        virtual float GetValue(Action* action);
+};
+
+class AttackFragmentMultiplier : public Multiplier
```

Comment:
Should be removed, this logic can be baked into the action/trigger in a one or two-liner